src/controllers/wallet/p2sh/tx-proposal.controller.js

```diff
@@ -56,6 +56,46 @@ async function buildTxProposal(req, res) {
   }
 }
 
+async function buildMintTokensTxProposal(req, res) {
+  const validationResult = parametersValidation(req);
+  if (!validationResult.success) {
+    res.status(400).json(validationResult);
+    return;
+  }
+
+  const {
+    token,
+    amount,
+  } = req.body;
+  const address = req.body.address || null;
+  const changeAddress = req.body.change_address || null;
+  const createAnotherMint = req.body.create_mint ?? true;
+  const mintAuthorityAddress = req.body.mint_authority_address || null;
+  const allowExternalMintAuthorityAddress = req.body.allow_external_mint_authority_address || null;
+
+  try {
+    if (changeAddress && !await req.wallet.isAddressMine(changeAddress)) {
+      throw new Error('Change address is not from this wallet');
+    }
+
+    const mintTokenTransaction = await req.wallet.prepareMintTokensData(
+      token,
+      amount,
+      {
+        address,
+        changeAddress,
+        createAnotherMint,
+        mintAuthorityAddress,
+        allowExternalMintAuthorityAddress,
+        signTx: false,
```

Comment:
This is apparently not working, is the signTx option on the next wallet-lib release?

Reply:
Yes, it will be available in the next wallet-lib release and I will come back here to bump the version, or open another PR just for it.